pages/IT/index.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="space">
+    <div class="dashboard-editor-container">
+      <tablo/>
+    </div>
+  </div>
+</template>
+
+
+<script>
+import PanelGroup from "@/components/Dashboard/PanelGroup.vue";
+import tablo from "@/components/Dashboard/ItTable-Installation.vue";
```

Comment:
You should use variables in English.

Reply:
Also, it's usually best to use the actual component name when importing them such as `ItTableInstallation` rather than a generic name.

Reply:
ItTable-Installation.vue  is changed ItTableInstallation.vue. And also tablo is not meaningful. It has been changed as ItTable.